target/linux/ath79/mikrotik/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -23,6 +23,9 @@ board=$(board_name)
 case "$FIRMWARE" in
 "ath9k-eeprom-ahb-18100000.wmac.bin")
 	case $board in
+	mikrotik,routerboard-2011)
+		caldata_mikrotik_ath9k 0x1000 0x440 $(macaddr_add "$mac_base" +5)
```

Comment:
As I do not have a Wi-Fi model handy I could not test this part.

Reply:
Works for me with this configuration. Old 19.07 release used MAC+11, though.